app/src/scenes/Health/Articles/AlcoolWithdrawalTreatment.js

```diff
@@ -0,0 +1,104 @@
+  return (
+    <NavigationWrapper title={title} timeReading={2}>
+      <TopContainer>
+        <P>
+          Dans cet article nous vous proposons une vulgarisation des concepts de symptômes de sevrage et un lexique des
+          principaux traitements utilisés dans le sevrage de la consommation d'alcool.
+        </P>
+        <P>
+          Il peut exister plusieurs <Bold>types de symptômes</Bold> lors de la réduction, ou de l'arrêt de la
+          consommation d'alcool :
+        </P>
+
+        <View className="flex flex-row">
+          <View className=" ">
+            <P noMarginBottom>
+              <Bold>{'   •  '}</Bold>
+            </P>
+          </View>
+          <View className="flex flex-row basis-11/12">
+            <P noMarginBottom>
+              <Bold>les symptômes de sevrage mineur</Bold> tel que agitation, nervosité ou des troubles du sommeil
+            </P>
+          </View>
+        </View>
+        <View className="flex flex-row">
+          <View>
+            <P noMarginBottom>
+              <Bold>{'   •  '}</Bold>
+            </P>
+          </View>
+          <View className="flex flex-row basis-11/12">
+            <P noMarginBottom>
+              <Bold>les symptômes de sevrage sévère</Bold> tel que les crises convulsives de sevrage (qui s'apparente à
+              une crise convulsive de l'épilepsie) ou d'autres manifestations décrites dans l'article dédié (
+              <Text className="underline" onPress={() => navigation.navigate('ALCOHOL_AND_DEPENDENCY')}>
+                article dépendance physique à l'alcool
+              </Text>
+              )
+            </P>
+          </View>
+        </View>
+        <View className="flex flex-row">
+          <View>
+            <P noMainBottom>
+              <Bold>{'   • '}</Bold> le <Bold>craving</Bold>
+            </P>
+          </View>
+        </View>
+        <Spacer size={20} />
+        <P className="flex flex-row basis-11/12">
+          Il existe des <Bold>traitements médicamenteux qui peuvent vous être proposés par un médecin</Bold> pour ces
+          trois grandes catégories de symptômes.
+        </P>
+        <P>
+          Ces symptômes apparaissent généralement entre <Bold>6 heures et 3 jours</Bold> et s'améliorent de manière
+          rapide sur <Bold>3 à 5 jours.</Bold>
+        </P>
+        <Spacer size={20} />
+        <P>
+          Il faut savoir que nous ne sommes pas tous égaux face à leur apparition et leur sévérité sans que ce ne soit
+          forcément lié à la quantité d'alcool que vous consommiez.
+        </P>
+        <P>
+          <Bold>
+            Aussi, si vous présentez des symptômes importants, que vous vous en inquiétez ou que vous avez une autre
+            pathologie chronique, n'hésitez pas à en parler à un professionnel de santé.
+          </Bold>{' '}
+          Les traitements ont vocation à soulager ces symptôme pour vous permettre de vous passer de l'alcool dans de
```

Comment:
Ah, pour une fois que je relis les articles et que ça sert à quelque chose ! J'étais frustré de ne rien voir dans l'article Craving :)

```suggestion
          Les traitements ont vocation à soulager ces symptômes pour vous permettre de vous passer de l'alcool dans de
```